client/src/components/Homepage/index.js

```diff
@@ -183,6 +207,18 @@ const Homepage = () => {
           <button className="nav-btn menu_btn" onClick={toggleMenu}>
             <h2>Menu</h2>
           </button>
+
+          <select 
+            name="color-theme" 
+            id="color-theme" 
+            className="nav-btn" 
+            onChange={(e)=>{setCurrentTheme(e.target.value)}}
+            style={{ display: visible ? 'block' : 'none' }}
+          >
+            <option value="cobalt" selected>Default</option>
```

Comment:
```suggestion
            <option value="cobalt" selected>Default Theme</option>
```
Can we make it `Default Theme` so that the user know this selector would change themes?